projects/core/src/occ/occ-models/occ-endpoints.model.ts

```diff
@@ -315,6 +315,30 @@ export interface OccEndpoints {
    * @member {string}
    */
   getVariantConfigurationOverview?: string;
+  /**
```

Comment:
It's better to move these endpoints to the CPQ library. So, in the library, we can have something like this:
```
import { OccEndpoint } from '@spartacus/core';

declare module '@spartacus/core' {
  interface OccEndpoints {
    ...
  }
}
```

Reply:
picked

Reply:
@Platonn As far as I recall, there was a reason to put the configurator related endpoints into this central file, we had discussed this for the first iteration also. Could you help me here?

Reply:
what is the reason?

Reply:
@Platonn Could you answer?

Reply:
I don't remember. Wei's suggestion feels right. We already follow that pattern in other libs:
![obraz](https://user-images.githubusercontent.com/4001059/115738436-582d1d80-a38d-11eb-85c9-23f57cdba0fd.png)

Reply:
Hmhh @Platonn , I remember that I had those endpoints the the feature lib for VC, and you told me to put them back a couple of months ago. I dimly recall something about extensibility.
But ok. Can be done.

Reply:
Done!